pdf_compression/README.rst

```diff
@@ -0,0 +1,115 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+PDF Compression & PDF/A
+=======================
+
+This addon provides the ability to convert outgoing and imported PDF attachments.
+You can compress the PDFs to save space in you database or file system.
+Also the possibility exists to convert PDFs in PDF/A. Currently implemented:
+
+* PDF/A-1b
+* PDF/A-2b (recommend to use this setting because it is faster and result in smaller files)
+
+
+Installation
+============
+
+#. You need at least Gostscript 9.18 (I recommend 9.19)
```

Comment:
You can put the ghostscript executable as external dependency in the manifest file.